src/azul/terra.py

```diff
@@ -161,7 +168,12 @@ def bq_name(self):
 
     def __str__(self) -> str:
```

Comment:
Missing doctest for https://github.com/DataBiosphere/azul/pull/2931#discussion_r674388377